### DISCUSSION TITLE
Campus expert application related issue

### DISCUSSION THREAD

**Discussion by Brijesh1963** [community/community#109731] [GitHub Education]
### Hi everyone,

My GitHub developer pack was approved but till now ,on the application section not able to apply for campus expert...
Please look at this matter, last date of apply 29th Feb.

![image](https://github.com/community/community/assets/98037970/529fe378-b64c-48c1-b0da-99f564a5e305)

**Comment by namith1003** [comment]
This means it's still under process and not yet approved, it's very misleading, I am in the same place, if you click on the benefit https://education.github.com/pack

it will say that the application is still under process, I have seen people that took 2 days to fully get approved and some lasting upto 8 or more days without a reply, there is not much we can do except maybe contacting educational support but patience is key here I guess.

Hope this answers your question.

**Reply by Brijesh1963** [reply]
ok
